src/Components/CreateImageWizard/steps/Oscap/components/useSelectorHandlers.tsx

```diff
@@ -0,0 +1,93 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import { useAppDispatch } from '../../../../../store/hooks';
+import { Filesystem, Services } from '../../../../../store/imageBuilderApi';
+import {
+  addKernelArg,
+  addPackage,
+  addPartition,
+  changeDisabledServices,
+  changeEnabledServices,
+  changeMaskedServices,
+  changeFileSystemConfigurationType,
+  clearKernelAppend,
+  clearPartitions,
+  removePackage,
+} from '../../../../../store/wizardSlice';
+import { parseSizeUnit } from '../../../utilities/parseSizeUnit';
+import { Partition, Units } from '../../FileSystem/FileSystemTable';
+
+export const useSelectorHandlers = () => {
```

Comment:
I like that this gets extracted out. Maybe we could separate these out into a single hook per handler function?

This would just mean that they would be even more re-usable and easier to test :)

Reply:
Thanks, that is a good idea, I will try to work on it. Still writing tests for this, so it does not get broken, will update you on the progress soon.